modem_setup.py

```diff
@@ -13,8 +13,13 @@
 
```

Comment:
<ugh> I accidently committed an old wifi password above. I'll move this script to examples + take the constants from the command line or `os.environ`